T&A Abandon notifications

thojou opened the pull request:
Hey all,

this PR is related to the Feature-Wiki Request https://docu.ilias.de/go/wiki/wpage_7663_1357
As requested, it removes the notification feature from the test module and drops unused language variables.

I'm looking forward to your Feedback.

Best Regards,
@thojou

Commit:
removes notifications feature and refactors tests

Commit:
deletes unused language variables and constants

thojou committed:
Remove left over additional information tags and drops unused language variables

kergomard commented:
Thank you very much for the PR @thojou !

I think there are some leftovers:
- in `ILIAS\Test\Settings\MainSettings\MainSettingsDatabaseRepository`, we should remove the corresponding values when instantiating `SettingsFinishing`.
- We should the corresponding columns in the database in an update step.

Otherwise this looks good to me.

Thanks again and best,
@kergomard

thojou committed:
Remove unused settings from database and query

thojou commented:
Hey @kergomard,

i removed the leftover query parts and implemented a database update step to remove the related columns from the database table. Also i removed arguments passed to the constructor.

Best Regards,
@thojou

kergomard commented:
Thank you very much @thojou !
